qa/supply-chain/audits.toml

```diff
@@ -966,6 +966,21 @@ criteria = "safe-to-deploy"
 delta = "0.8.1 -> 0.9.1"
 notes = "The ECC core team maintains this crate, and we have reviewed every line."
 
+[[audits.zcash_primitives]]
+who = "Jack Grigg <jack@electriccoin.co>"
+criteria = ["safe-to-deploy", "crypto-reviewed"]
+delta = "0.9.1 -> 0.10.0"
+notes = "The ECC core team maintains this crate, and we have reviewed every line."
+
+[[audits.zcash_primitives]]
+who = "Jack Grigg <jack@electriccoin.co>"
+criteria = ["safe-to-deploy", "crypto-reviewed"]
+delta = "0.10.0 -> 0.10.1"
```

Comment:
There needs to be an audit delta for 0.10.2.

Reply:
That cannot happen until `zcash_primitives 0.10.2` is published.